src/styles/_card.scss

```diff
@@ -0,0 +1,120 @@
+.catalog {
+  display: grid;
+  justify-items: center;
+  grid-template-columns: 1fr;
+  grid-gap: 48px 46px;
+  padding: 50px 40px;
+
+  @media (min-width: 488px) {
+    grid-template-columns: repeat(2, 1fr);
+  }
+
+  @media (min-width: 768px) {
+    grid-template-columns: repeat(3, 1fr);
+  }
+
+  @media (min-width: 1024px) {
+    grid-template-columns: repeat(4, 1fr);
+  }
+}
+
+.catalog-card {
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
+  width: 200px;
```

Comment:
The fixed width of 200px for the .catalog-card might cause layout issues on smaller screens. Consider using a relative unit or a percentage to make it more responsive.